helmfile.d/config/mongo-express-common.yaml.gotmpl

```diff
@@ -0,0 +1,18 @@
+environment: {{ .Environment.Name }}
+
+image:
+  tag: v0.1.0
```

Comment:
I'd remove this, and let the chart values.yaml decide the image tag

Reply:
ack